app/mailers/user_mailer.rb

```diff
@@ -6,15 +6,6 @@
 # See MIT-LICENSE file or http://www.opensource.org/licenses/mit-license.php
 #------------------------------------------------------------------------------
 class UserMailer < ActionMailer::Base
-  def password_reset_instructions(user)
```

Comment:
Worth adding that to the migration guide notes on the off chance anyone customised these

Reply:
Done.